packages/dds/shared-object-base/src/serializer.ts

```diff
@@ -84,12 +88,14 @@ export class FluidSerializer implements IFluidSerializer {
 	 *
 	 * Any unbound handles encountered are bound to the provided IFluidHandle.
 	 */
-	public encode(input: any, bind: IFluidHandle) {
+	// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any -- TODO: ddsFuzzHarness breaks
```

Comment:
Should we file a work item to tackle this?

Reply:
Done. I filed it for updating the types; fixing ddsFuzzHarness will have to come as a consequence when we do it.